tests/MongoDBDriverManager_executeQuery_error-001.phpt

```diff
@@ -0,0 +1,60 @@
+--TEST--
+PHPC-924: Cursor::setTypeMap() may unnecessarily convert first BSON document (type map)
```

Comment:
Shouldn't this have [HHVM-288](https://jira.mongodb.org/browse/HHVM-288) if it's included? That ticket was also resolved as Won't Fix so I'm not sure why this is being pulled in.